src/components/DownloadBlock/index.tsx

```diff
@@ -1,23 +1,42 @@
 import Badge from '@app/components/Common/Badge';
+import { Permission, useUser } from '@app/hooks/useUser';
 import type { DownloadingItem } from '@server/lib/downloadtracker';
 import { defineMessages, FormattedRelativeTime, useIntl } from 'react-intl';
 
 const messages = defineMessages({
   estimatedtime: 'Estimated {time}',
+  seasonepisodenumber: ': Season {seasonNumber} Episode {episodeNumber}',
 });
 
 interface DownloadBlockProps {
   downloadItem: DownloadingItem;
   is4k?: boolean;
+  title?: string;
 }
 
-const DownloadBlock = ({ downloadItem, is4k = false }: DownloadBlockProps) => {
+const DownloadBlock = ({
+  downloadItem,
+  is4k = false,
+  title,
+}: DownloadBlockProps) => {
   const intl = useIntl();
+  const { hasPermission } = useUser();
+
+  const formattedTitle: string =
+    title +
+    intl.formatMessage(messages.seasonepisodenumber, {
+      seasonNumber: downloadItem?.episode?.seasonNumber,
+      episodeNumber: downloadItem?.episode?.episodeNumber,
+    });
```

Comment:
```suggestion
  const formattedTitle =
    intl.formatMessage(messages.seasonepisodenumber, {
    title,
      seasonNumber: downloadItem?.episode?.seasonNumber,
      episodeNumber: downloadItem?.episode?.episodeNumber,
    });
```

And then update the i18n string to have {title} in it